src/JobPanel/backend/service_base.py

```diff
@@ -30,15 +31,13 @@ class ServiceStatus(enum.IntEnum):
 
 
 
-def LongRequest():
+def LongRequest(func):
```

Comment:
Upravil jsem, původní nefungovalo, ještě by bylo možné udělat složitější variantu, kde by se vytvořila nová funkce, která by uvnitř volala původní a u té nové by se nastavil příznak a vrátila by se.